GVRf/Framework/src/org/gearvrf/scene_objects/GVRWebViewSceneObject.java

```diff
@@ -1,112 +0,0 @@
-/* Copyright 2015 Samsung Electronics Co., LTD
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.gearvrf.scene_objects;
-
-import org.gearvrf.GVRContext;
-import org.gearvrf.GVRDrawFrameListener;
-import org.gearvrf.GVRExternalTexture;
-import org.gearvrf.GVRMaterial;
-import org.gearvrf.GVRMesh;
-import org.gearvrf.GVRSceneObject;
-import org.gearvrf.GVRTexture;
-import org.gearvrf.GVRMaterial.GVRShaderType;
-
-import android.graphics.Canvas;
-import android.graphics.SurfaceTexture;
-import android.util.Log;
-import android.view.Surface;
-import android.webkit.WebView;
-
-/**
```

Comment:
rather than removing the class outright, please deprecate it first.  also, note in the javadoc to reference the GVRViewSceneObject and the example in the scene-objects sample.  

it is generally more polite to deprecate an api/class before removing it.  we'll remove it in a subsequent release.